Document/0x06a-Platform-Overview.md

```diff
@@ -112,24 +107,25 @@ An IPA file has a built-in directory structure. The example below shows this str
 
 #### IPA Payloads - A Closer Look
 
-Let’s take a closer look at the different files that are to be found in the ZIP compressed IPA container. It is necessary to understand that this is the raw architecture of the bundle container and not the definitive form after installation on the device. It uses a relatively flat structure with few extraneous directories in an effort to save disk space and simplify access to the files. The bundle contains the application executable and any resources used by the application (for instance, the application icon, other images, and localized content) in the top-level bundle directory.
+Let us take a closer look at the different files that can be found in the IPA container. It is necessary to understand that this is the raw architecture of the bundle container and not the definitive form after its installation on a device. It uses a relatively flat structure with few extraneous directories in an effort to save disk space and simplify access to the files. The bundle contains the application executable and any resources used by the application (for instance, the application icon, other images, and localized content) in the top-level bundle directory.
 
-- **MyApp**: The executable containing the application’s code, which is compiled and not in a ‘readable’ format.
+- **MyApp**: The executable containing the application code, which is compiled, not in a ‘readable’ format.
 - **Application**: Icons used at specific times to represent the application.
-- **Info.plist**: Containing configuration information, such as its bundle ID, version number, and display name.
+- **Info.plist**: Configuration information, such as its bundle ID, version number, and application display name.
 - **Launch images**: Images showing the initial interface of the application in a specific orientation. The system uses one of the provided launch images as a temporary background until the application is fully loaded.
-- **MainWindow.nib**: Contains the default interface objects to load at application launch time. Other interface objects are then either loaded from additional nib files or created programmatically by the application.
-- **Settings.bundle**: Contains any application-specific preferences using property lists and other resource files to configure and display them.
-- **Custom resource files**: Non-localized resources are placed at the top level directory and localized resources are placed in language-specific subdirectories of the application bundle. Resources consist of nib files, images, sound files, configuration files, strings files and any other custom data files you need for your application.
+- **MainWindow.nib**: Default interface objects to load at the application launch time. Other interface objects are then either loaded from additional nib files or created programmatically by the application.

```

Comment:
> load at the application launch time

I think "load at application launch time" is correct, same way that you would say "we'll have wine at dinner time", and not "we'll have wine at the dinner time".